lib/android-helpers.js

```diff
@@ -400,13 +401,18 @@ helpers.installOtherApks = async function (otherApps, adb, opts) {
 helpers.initUnicodeKeyboard = async function (adb) {
   logger.debug('Enabling Unicode keyboard support');
   logger.debug("Pushing unicode ime to device...");
-  await adb.install(unicodeIMEPath, {replace: false});
+  try {
+    await adb.install(unicodeIMEPath, {replace: false});
+  } catch (err) {
+    await adb.uninstallApk(UNICODE_IME_PKG_ID);
```

Comment:
I think a log message here saying we're retrying would be helpful for reading the logs during debugging

Reply:
uninstallApk does write logs anyway, so I thought it might too much

Reply:
sure, i just meant for us to understand why we'd see an install, then an uninstall, then another install. presumably the existing logs don't tell us that there's a retry going on. but it's not a big deal either way.